modules/phase_field/src/ics/PolycrystalColoringIC.C

```diff
@@ -36,8 +36,15 @@ PolycrystalColoringIC::PolycrystalColoringIC(const InputParameters & parameters)
 Real
 PolycrystalColoringIC::value(const Point & p)
 {
+  _poly_ic_uo.getGrainsBasedOnPoint(p, _grains);
+
+  if (_grains.empty())
+    return 0.0;
+
+  // If we have several active grains, we may want to do something different with the OPs here
+  auto grain_id = _grains[0];
```

Comment:
For now, I'm just arbitrarily grabbing the first grain and using that to figure out which OP to use, then of course the value.